_sass/module/_candidate.scss

```diff
@@ -1,9 +1,23 @@
 $candidate-photo-width: 10rem;
 
-.candidate__occupation {
+.candidate__occupation__commitee_name {
   @include heading('h3');
   color: $subheading-color;
   font-style: italic;
+  font-size: 18px;
+  margin-bottom: 5px;
+}
+
+.candidate__expenditure__ceiling {
+  color: $subheading-color;
+  font-style: italic;
+  font-size: 14px;
+}
+
+.candidate__expenditure__ceiling a {
```

Comment:
This is [Sass](https://sass-lang.com/), so you should merge this rule with the one above, e.g.

```scss
.candidate__expenditure-ceiling {
  // properties...

  a  {
  // properties...
  }
}